test/methods.jl

```diff
@@ -268,37 +268,37 @@ end
     exp1 = Raster([0.1 0.2; 0.3 0.4], (X(Sampled([2.0, 3.0]; span=span_x1)), Y([5.0, 6.0])))
     exp2 = Raster([1.1 1.2; 1.3 1.4], (X(Sampled([3.0, 4.0]; span=span_x2)), Y([6.0, 7.0])))
     @test val(span(mosaic(first, exp1, exp2), X)) == [1.5 2.5 3.5; 2.5 3.5 4.5]
-    @test all(mosaic(first, [reg1, reg2]) .=== 
+    @test all(mosaic(first, [reg1, reg2]) .===
               mosaic(first, irreg1, irreg2) .===
-              mosaic(first, (irreg1, irreg2)) .=== 
-              [0.1 0.2 missing; 
-               0.3 0.4 1.2; 
+              mosaic(first, (irreg1, irreg2)) .===
+              [0.1 0.2 missing;
+               0.3 0.4 1.2;
                missing 1.3 1.4])
     @test all(mosaic(last, reg1, reg2) .===
               mosaic(last, irreg1, irreg2) .===
-              mosaic(last, exp1, exp2) .=== [0.1 0.2 missing; 
-                                             0.3 1.1 1.2; 
+              mosaic(last, exp1, exp2) .=== [0.1 0.2 missing;
+                                             0.3 1.1 1.2;
                                              missing 1.3 1.4])
 
-    @test all(mosaic(first, [reverse(reg2; dims=Y), reverse(reg1; dims=Y)]) .=== 
-              [missing 0.2 0.1; 
```

Comment:
Try not to change anything except what you're working on in a PR

Reply:
That was a complete accident sorry! I should have spotted it.